app/src/main/java/io/neurolab/MainActivity.java

```diff
@@ -14,19 +14,28 @@
 import android.view.Menu;
 import android.view.MenuItem;
 import android.view.View;
-
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
+import android.view.animation.TranslateAnimation;
+import android.widget.ImageView;
 import io.neurolab.settings.FeedbackSettings;
 
 public class MainActivity extends AppCompatActivity
         implements NavigationView.OnNavigationItemSelectedListener {
 
+    private ImageView rocketimage;
+    private int lastPos = 0;
+    private int newPos = -300;
+    private boolean moving;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
         Toolbar toolbar = findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
 
+        rocketimage = findViewById(R.id.rocketimage);
         FloatingActionButton fab = findViewById(R.id.fab);
```

Comment:
Travis build is failing because of this. Have you at-least run this code and installed on mobile phone before making the PR?

Reply:
@CloudyPadmal yes sir. Take a look at the gif in #28 . It is the working prototype on my device only.

Reply:
This is too much conversation. Why can't you fix it? The build is failing and why are you justifying it? Why do you need to merge every PR into other PRs?

This is basic Android development. If a resource referred in a source file has been removed the build fails. Please do some research and fix it. Thanks.

Reply:
This issue is not resolved

Reply:
@CloudyPadmal give me some time I am looking into it. There is no build failure at my side. But since the Travis build is failing I am definitely looking into it. Thank you.